File: app/src/main/java/chat/hola/com/app/comment/model/Response.java
package chat.hola.com.app.comment.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;
/**
 * <h1>Response</h1>
 *
 * @author 3embed
 * @since 4/9/2018
 * @version 1.0
 */

public class Response {
    @SerializedName("code")
    @Expose
    private Integer code;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("data")
    @Expose
    private List<Comment> comments = null;

    public Integer getCode() {
        return code;
    }

    public void setCode(Integer code) {
        this.code = code;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Comment> getData() {
        return comments;
    }

    public void setData(List<Comment> data) {
        this.comments = data;
    }

}

File: remote/src/main/java/com/kotlintestgradle/remote/model/common/Response.kt
package com.kotlintestgradle.remote.model.common

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author 3Embed
 *used for retrofit
 * @since 1.0 (23-Aug-2019)
 */
data class Response(
    @Expose @SerializedName("Success") var success: Boolean = false,
    @Expose @SerializedName("message") var message: String? = null,
    @Expose @SerializedName("errorCode")
    var errorCode: String? = null
)
